### PR TITLE
feat: Add support for environments when updating content types

### PR DESCRIPTION
**Pull request by alexandradeas**
Add support for updating a specific environment when updating a content type

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/davecgh/go-spew v1.1.0 // indirect
```
Comment (Ardjan-Aalberts) [inline go.mod:6]: Please run `make lint` to get the latest versions of the dependencies.

File: go.sum [hunk 2]
```diff
@@ -0,0 +1,43 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```
Comment (Ardjan-Aalberts) [inline go.sum:1]: Could you remove the go.sum file? Since this is generated we don't save this in the repository.

**Comment by Ardjan-Aalberts** [comment]
Hi @alexandradeas thanks for taking the time to create this PR! I would like to merge your pull request as soon as possible. I added two small comments but the code looks great as far as I can see!

File: go.sum (hunk 2 above)
Comment (alexandradeas) [reply go.sum]: removed, is it worth adding this to the gitignore or would this mess with releases?

**Review by Ardjan-Aalberts — APPROVED** [review]

**Pull request merged**